src/styles/main.scss

```diff
@@ -1,3 +1,70 @@
+$days-in-month: 31;
+$month-length: 28;
+
 body {
   margin: 0;
 }
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+}
+
+.calendar {
+  gap: 1px;
+  display: flex;
+  flex-wrap: wrap;
+  width: 706px;
+  padding: 10px;
+}
+
+$days-margin:
+  "mon" 0,
+  "tue" 101px,
```

Comment:
you can calculate it according to the day index by using 2 variables (`$day-size` and `$days-margin`)